sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
+
+/// Specifies an acceptable orientation for the interface to be in
+/// A mask of acceptable interface orientations is created by or'ing individual
+/// options
+enum /* options */ InterfaceOrientation {
+  Portrait  = 1,
+  Landscape = 2,
+};
+
+/// Specifies a particular device orientation
+/// A mask of device interface orientations is created by or'ing individual
+/// options
+enum /* options*/ DeviceOrientation {
```

Comment:
I would order these as PortraitUp, LandscapeLeft, PortraitDown, LandscapeRight.
I am not sure what the names should be.
Maybe document it as:
```
/// To determine which values correspond to which orientations, first position the device
/// in its default orientation (this is the orientation that the system first uses for its
/// boot logo, or the orientation in which the hardware logos or markings are upright, or
/// the orientation in which the cameras are at the top). If this is a portrait orientation,
/// then this is PortraitUp. Otherwise, it's LandscapeLeft.
/// As you rotate the device by 90 degrees in a counter-clockwise direction around the
/// axis that traverses the screen, you step through each value in this enum in the order
/// given. (For a device with a landscape default orientation, the orientation obtained by
/// rotating the device 90 degrees clockwise from its default orientation is PortraitUp.)
```